pkgs/applications/misc/wmenu/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromSourcehut
+, pkg-config
+, cmake
+, meson
+, ninja
+, cairo
+, pango
+, wayland
+, wayland-scanner
+, wayland-protocols
+, libxkbcommon
+, scdoc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wmenu";
+  version = "0.1.2";
+
+  src = fetchFromSourcehut {
+    owner = "~adnano";
+    repo = "wmenu";
+    rev = version;
+    hash = "sha256-mS4qgf2sjgswasZXsmnbIWlqVv+Murvx1/ob0G3xsws=";
+  };
+
+  patches = [
+    (fetchpatch {
```

Comment:
sorry, i didnt include in my patch, but you need to add fetchpatch to list of arguments to this file

Reply:
sorry i didnt say this earlier but maybe we should add some comment on why we have this patch